app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -77,6 +77,14 @@ class StorySummaryViewModel(
           ephemeralChapterSummary.chapterSummary.title,
           ephemeralChapterSummary.writtenTranslationContext
         ),
+        previousChapterTitle = if (index > 0) {
+          translationController.extractString(
+            ephemeralStorySummary.chaptersList[index - 1].chapterSummary.title,
+            ephemeralStorySummary.chaptersList[index - 1].writtenTranslationContext
+          )
+        } else {
+          null
+        },
```

Comment:
```suggestion
        } else null,
```

Small change to make it a bit more compact.

Reply:
This has been done too.